Kong/docs.konghq.com
# Feat: decK gateway CLI

### Description

The decK gateway interaction commands were moved into a `gateway` subcommand in decK version 1.28.0, which also included some refactoring in how they function. At the time, we avoided updating the docs as users were unlikely to have a version of decK that uses those commands, and they were not yet stable. 
They are now ready to be updated, and the docs are causing more confusion by not using `deck gateway` in examples.

Took two different approaches re: versioning these commands:
* In the decK documentation (docs.konghq.com/deck), I version-tagged the commands whenever they were used in code blocks. If they were mentioned in text, I removed the `deck` portion.
  * Duplicated one file (docker run reference), as the whole page would've needed version tagging otherwise.
* In the rest of the site, I didn't version the commands, as there's no clear way to know what version of decK someone is using with a specific version of Gateway. All of the Gateway, Konnect, and plugin docs just use the `deck gateway` commands.

https://konghq.atlassian.net/browse/DOCU-3565
https://konghq.atlassian.net/browse/DOCU-3545

### Testing instructions

Preview link: Spot check some of the changed pages:  https://deploy-preview-7585--kongdocs.netlify.app/deck/latest/guides/getting-started/

### Checklist
